src/vaf.nim

```diff
@@ -23,6 +24,9 @@ let p = newParser("vaf - very advanced fuzzer"):
   flag("-pu", "--printurl", help="prints the url that has been requested")
 
 try:
+
+    # discard initVafConfig()
```

Comment:
this code creates the file .vaf.json in the user directory if it does not exist and writes {} to it. not sure if i should keep it here or not

Reply:
Meaby .vaf file without  the .json extention cause its not nessaresry but ill look at it later